tests/integration/test_smart_contracts.py

```diff
@@ -147,6 +147,30 @@ def test_get_contract_with_contract_id(self):
         self.assertEqual(response.get("status"), 200, response)
         jsonschema.validate(response.get("response"), schema.smart_contract_at_rest_schema)
 
+    def test_get_contract_logs_with_contract_id(self):
+        response = self.client.get_smart_contract_logs(smart_contract_id=SMART_CONTRACT_ARGS_ID)
+        self.assertTrue(response.get("ok"), response)
+        self.assertEqual(response.get("status"), 200)
+        jsonschema.validate(response.get("response"), schema.smart_contract_logs_schema)
+
+    def test_get_contract_logs_with_contract_id_and_tail(self):
+        response = self.client.get_smart_contract_logs(smart_contract_id=SMART_CONTRACT_ARGS_ID, tail=100)
+        self.assertTrue(response.get("ok"), response)
+        self.assertEqual(response.get("status"), 200)
+        jsonschema.validate(response.get("response"), schema.smart_contract_logs_schema)
+
+    def test_get_contract_logs_with_contract_id_and_since(self):
+        response = self.client.get_smart_contract_logs(smart_contract_id=SMART_CONTRACT_ARGS_ID, since="2019-09-16T23:10:04.000Z")
+        self.assertTrue(response.get("ok"), response)
+        self.assertEqual(response.get("status"), 200)
+        jsonschema.validate(response.get("response"), schema.smart_contract_logs_schema)
+
+    def test_get_contract_logs_with_contract_id_tail_and_since(self):
+        response = self.client.get_smart_contract_logs(smart_contract_id=SMART_CONTRACT_ARGS_ID, tail=100, since="2019-09-16T23:10:04.000Z")
+        self.assertTrue(response.get("ok"), response)
+        self.assertEqual(response.get("status"), 200)
+        jsonschema.validate(response.get("response"), schema.smart_contract_logs_schema)
+
```

Comment:
What about a test with a bad contract id? We should assert on fail-cases like a 404's for bad smart contract id as well